openmls/src/treesync/diff.rs

```diff
@@ -466,13 +487,21 @@ impl<'a> TreeSyncDiff<'a> {
     /// Helper function computing the resolution of a node with the given index.
     /// If an exclusion list is given, do not add the public keys of the leaves
     /// given in the list.
+    ///
+    /// Returns The list of HPKE public keys.
+    /// In case node_id is not in the tree a LibraryError is returned.
```

Comment:
And again. At this point, why not have an error that is either a `NotInTree` error or a `LibraryError`?